src/Datadog.Trace.ClrProfiler.Managed/Datadog.Trace.ClrProfiler.Managed.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
We keep switching back and forth between having and not having the UTF-8 BOM. Why does the tool even modify this file?

Reply:
The Versions replacement logic does this. You suggested we add this option to the versioning tool when it was implemented: `new UTF8Encoding(encoderShouldEmitUTF8Identifier: false)`

Reply:
Resolving this comment because [duplicate](https://github.com/DataDog/dd-trace-dotnet/pull/624#discussion_r374954319). 😉